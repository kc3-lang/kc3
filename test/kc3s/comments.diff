--- comments.out.expected	2025-03-23 13:05:12.237823198 +0100
+++ comments.out	2025-03-23 13:11:11.112361758 +0100
@@ -1 +1,5 @@
+env_eval_ident: unbound ident: kc3
+env_eval_ident: stacktrace:
+Stacktrace:
+
 ok
