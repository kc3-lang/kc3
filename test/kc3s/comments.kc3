#!/usr/bin/env kc3s
# -*- elixir -*-
# kc3
# Copyright from 2020 to 2025 kmx.io <contact@kmx.io>
puts("ok")
