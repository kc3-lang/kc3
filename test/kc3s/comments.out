env_eval_ident: unbound ident: kc3
env_eval_ident: stacktrace:
Stacktrace:

ok
